_pkgdown.yml

```diff
@@ -1,6 +1,7 @@
 url: https://pharmaverse.github.io/admiral
 
 template:
+  math-rendering: mathjax
```

Comment:
is this intentional?  Just checking!?! :)

Reply:
Yes, I rendered the documentation with pkgdown 2.1.1 and noticed that formulas are no longer rendered correctly. Adding this line fixes the issues.